### ISSUE TITLE
Add quartz.threadPool.threadCount

### ISSUE THREAD

**Issue by orbiran88** [jjethwa/rundeck#148]
Hi, 
I thinks that add the "quartz.threadPool.threadCount" to rundeck-config.properties would be great... and control the value with ENV variable, what do you think?

**Comment by jjethwa** [comment]
Hi @orbiran88 

I left most of the advanced configurations off as users can use their own properties file by using the /etc/rundeck volume.  We can certainly add the ability to set quartz.threadPool.threadCount though.  Would you like to submit a PR?

**Comment by orbiran88** [comment]
Yes I would like to submit PR, do I need to get permission from you to submit PR to master\dev?

**Comment by jjethwa** [comment]
Hi @orbiran88 

Please open it against master.  Thanks! 😄
